usr.bin/look/look.c

```diff
@@ -132,21 +138,35 @@ main(int argc, char *argv[])
 
 	match = 1;
 
-	do {
-		if ((fd = open(file, O_RDONLY, 0)) < 0 || fstat(fd, &sb))
+#ifndef WITHOUT_CAPSICUM
+	cap_rights_init(&rights, CAP_MMAP_R, CAP_READ, CAP_FSTAT);
+#endif
+	int fds[argc > 1 ? argc - 1 : argc];
```

Comment:
We don't use dynamic stack variable initialization from newer C. You should use malloc or calloc.

Reply:
I didn't find anything in `style(9)` explicitly disallowing VLAs, and technically they’re not from 'newer' C—VLAs have been part of the standard since C99 and are mandatory again in C23 (after being optional since C11.) I believe using a VLA here would simplify our approach significantly and avoid unnecessary heap allocation, improving memory efficiency. I’d like to request reconsideration for this specific case, but if there’s a strong rationale against VLAs in general, I'd appreciate any guidance on it.

Reply:
Yes, you are right. However, I (maybe I should say We as project) try to avoid them. Stack is limited and I don't want to even analyze if we won't exceed it on all supported platforms or if the command line lenght limit will be enough.

Reply:
All right, it makes sense.